packages/components/src/components/code-snippet/_code-snippet.scss

```
@@ -302,9 +303,27 @@
   }
 
   .#{$prefix}--snippet-button .#{$prefix}--btn--copy__feedback {
-    top: rem(25px);
+    top: rem(
+      50.8px
+    ); // (The height of button) + (The height of the tooltip's triangle) + 4px
     left: 50%;
     right: auto;
+
+    &:before {
```

should we use the CSS3 selector with double colon instead?

```suggestion
    &::before {
```